audio-worklet/noisegate-test.html

```diff
@@ -62,7 +62,7 @@
         // The signal should be all 1's prior to timeToThreshold to simplify
         // the math that relates the closing index to the timeConstant.
         let gain = new GainNode(context, {gain : 1});
-        let constantSource = new ConstantSourceNode(context, {offset: 1});
+        let constantSource = new ConstantSourceNode(context, {gain: 1});
```

Comment:
"Be consistent: Line 64 has "gain : 1", but line 65 has "offset: 1". I think the preferred style is "gain: 1"."

This was in reference to a comment you made earlier to be consistent in using gain in both cases, but maybe I misunderstood it?